### PR TITLE
Windows: add build scripts for the installer

### PR DESCRIPTION
**Pull request by compnerd**
This adds the installer packaging rules to allow building the Swift
installer which collects all the MSIs.

### CODE REVIEW

**Comment by compnerd** [comment]
CC: @shahmishal

File: platforms/Windows/installer.wxs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:bal="http://schemas.microsoft.com/wix/BalExtension">
+  <Bundle Name="Swift" Version="$(var.ProductVersion)" Manufacturer="swift.org" UpgradeCode="8c75f32a-7bdf-4c61-abf6-c7e1c4b8fbf6">
+    <BootstrapperApplicationRef Id="WixStandardBootstrapperApplication.HyperlinkLicense">
+      <bal:WixStandardBootstrapperApplication LicenseUrl="" LogoFile="Resources/swift.png" SuppressOptionsUI="yes" SuppressRepair="no" />
```
Comment (shahmishal) [inline platforms/Windows/installer.wxs:5]:
```
Resources/swift.png
```
Where is this image comping from?

Comment (compnerd) [reply platforms/Windows/installer.wxs:5]: I yanked it from swift.org

Comment (compnerd) [reply platforms/Windows/installer.wxs:5]: I should mention that if you have a better way to get the project logo, I'm happy to use that, I don't really have any issue with adjusting the content.

Comment (shahmishal) [reply platforms/Windows/installer.wxs:5]: Where is the Resources directory located, is it in swift-installer-scripts repo or another location?

Comment (compnerd) [reply platforms/Windows/installer.wxs:5]: It is in the same repository, it is being created as part of this commit.

Comment (shahmishal) [reply platforms/Windows/installer.wxs:5]: https://developer.apple.com/swift/resources/ at the bottom of the page it contains logos.

Comment (shahmishal) [reply platforms/Windows/installer.wxs:5]: Are you able to use svg format?

Comment (compnerd) [reply platforms/Windows/installer.wxs:5]: Thanks, thats where I originally grabbed the image from.  I don't think that the SVG format works for this, which is why I have the png.

**Review by shahmishal — APPROVED** [review]

**Pull request merged**